qml/Ping360ControlPanel.qml

```diff
@@ -42,13 +33,83 @@ Item {
                 control.to: 100
                 control.onMoved: ping.range = control.value
             }
-            PingSlider {
-                Layout.fillWidth: true
-                text: "Receiver Gain"
-                value: ping.gain_setting
-                control.from: 0
-                control.to: 2
-                control.onMoved: ping.gain_setting = control.value
+
+            // Uncomment to debug ComboSlider below
+//            PingSlider {
+//                Layout.fillWidth: true
+//                text: "Receiver Gain"
+//                value: ping.gain_setting
+//                control.from: 0
+//                control.to: 2
+//                control.onMoved: ping.gain_setting = control.value
+//            }
+
+            RowLayout {
+                Layout.preferredHeight: 50
+
+                Label {
+                    horizontalAlignment: Text.AlignRight
+                    wrapMode: Text.Wrap
+                    Layout.fillWidth: true
+                    Layout.preferredWidth: 120
+                    text: "Receiver Gain: "
+                }
+
+                ComboSlider {
+                    Layout.fillWidth: true
+                    value: ping.gain_setting
+                    model: ["Low", "Medium", "High"]
+                    onOptionChanged: {
+                        console.log("Value " + value + " at key " + key)
+                        ping.gain_setting = key
+                    }
+                }
+
+                Label {
+                    Layout.fillWidth: true
+                    Layout.preferredWidth: 40
+                    text: ping.gain_setting
+                }
+            }
+
+            // Uncomment to debug ComboSlider below
+//            PingSlider {
+//                text: "Sector Angle (degrees)"
+//                value: ping.sectorSize
+//                control.from: 30
+//                control.to: 360
+//                control.stepSize: 15
+//                control.snapMode: Slider.SnapAlways
+//                control.onMoved: ping.sectorSize = control.value
+//            }
+
+            RowLayout {
+                Layout.preferredHeight: 50
+
+                Label {
+                    horizontalAlignment: Text.AlignRight
+                    wrapMode: Text.Wrap
+                    Layout.fillWidth: true
+                    Layout.preferredWidth: 120
+                    text: "Sector Angle (degrees): "
+                }
+
+                ComboSlider {
+                    Layout.fillWidth: true
+                    // value: ping.sectorSize // doesn't work
+                    value: 6 // initial 'value' is 360
+                    model: [30, 45, 60, 90, 180, 360]
+                    onOptionChanged: {
+                        console.log("Value " + value + " at key " + key)
```

Comment:
This log does not help us to debug it in the end, is this really necessary here or is it a development code ?